tests/testStreamOpen.cpp

```diff
@@ -331,6 +331,17 @@ TEST_F(StreamOpenOutput, PlaybackFormatFloatReturnsErrorBeforeLollipop){
     }
 }
 
+TEST_F(StreamOpenOutput, PlaybackFormatFloatReturnsFloatBeforeLollipopWithFormatConversionAllowed){
```

Comment:
PlaybackFormatFloatReturnsFloatWithFormatConversionAllowed always, not just on Lollipop

Reply:
See previous comments.

Reply:
We should add a second test that does the exact same thing but for SDK >= L.